resync/api/monitoring_dashboard.py

```
@@ -244,70 +199,66 @@ async def _check_alerts_redis(self, sample: MetricSample) -> None:
             })
 
         if new_alerts:
             redis = get_redis_client()
             pipe = redis.pipeline()
             for alert in new_alerts:
                 pipe.lpush(REDIS_KEY_ALERTS, json_dumps(alert))
-            pipe.ltrim(REDIS_KEY_ALERTS, 0, 19) # Manter últimos 20
+            pipe.ltrim(REDIS_KEY_ALERTS, 0, 19)  # Manter últimos 20
             try:
                 await pipe.execute()
             except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)